### PR TITLE
feat: extend animal list

### PR DESCRIPTION
**Pull request by gegaryfa**

### CODE REVIEW

File: .idea/workspace.xml
Comment (anandvarma) [inline .idea/workspace.xml:1]: Get rid of this.

File: dictionary.go [hunk 1]
```diff
@@ -304,244 +304,530 @@ var colors = []string{
 
 // Word corpus of animals.
 var animals = []string{
+	"aardvark",
 	"albatross",
 	"alligator",
 	"alpaca",
 	"anchovy",
-	"ant",
```
Comment (anandvarma) [inline dictionary.go:311]: I see a couple of existing entries deleted, any particular reason behind this?

**Review by anandvarma — CHANGES_REQUESTED** [review]
Thanks for the PR, some initial comments inline.

File: dictionary.go (hunk 1 above)
Comment (gegaryfa) [reply dictionary.go:311]: not really. I only grabbed a list of animals I found online and replaced the existing list with it. I will merge the old and new lists

**Review by anandvarma — APPROVED** [review]

**Pull request merged**